resources/views/create.blade.php

```diff
@@ -6,40 +6,90 @@
 
     <div class="container-xl">
 
+        @if(!in_array("published_at", array_column($columns->toArray(), 'name')))
         <div class="card">
-
-            <{{ $modelJSName }}-form
-                :action="'{{'{{'}} url('admin/{{ $resource }}') }}'"
-                @if($hasTranslatable):locales="@{{ json_encode($locales) }}"
-                :send-empty-locales="false"@endif
-
-                inline-template>
-
-                <form class="form-horizontal form-create" method="post" {{'@'}}submit.prevent="onSubmit" :action="this.action" novalidate>
-
-                    <div class="card-header">
-                        <i class="fa fa-plus"></i> {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.create') }}
-                    </div>
-
-                    <div class="card-body">
-
-                        {{'@'}}include('admin.{{ $modelDotNotation }}.components.form-elements')
-
+        @endif
+
+        <{{ $modelJSName }}-form
+            :action="'{{'{{'}} url('admin/{{ $resource }}') }}'"
+            @if($hasTranslatable):locales="@{{ json_encode($locales) }}"
+            :send-empty-locales="false"@endif inline-template>
+
+            <form class="form-horizontal form-create" method="post" {{'@'}}submit.prevent="onSubmit" :action="this.action" novalidate>
+                @if(in_array("published_at", array_column($columns->toArray(), 'name')))
+
+                <div class="row">
+                    <div class="col">
+                        <div class="card">
+                            <div class="card-header">
+                                <i class="fa fa-plus"></i> {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.create') }}
+                            </div>
+                            <div class="card-body">
+                                {{'@'}}include('admin.{{ $modelDotNotation }}.components.form-elements')
+                            </div>
+                        </div>
                     </div>
 
-                    <div class="card-footer">
-	                    <button type="submit" class="btn btn-primary" :disabled="submiting">
-		                    <i class="fa" :class="submiting ? 'fa-spinner' : 'fa-download'"></i>
-		                    @{{ trans('brackets/admin-ui::admin.btn.save') }}
-	                    </button>
+                    <div class="col-md-12 col-lg-12 col-xl-5 col-xxl-4">
```

Comment:
if this is a separate card, then make it as a card and add it to another file, not to create/edit blades.